tests/dispatch_test.c

```diff
@@ -33,6 +33,8 @@
 #if __has_include(<sys/event.h>)
 #define HAS_SYS_EVENT_H 1
 #include <sys/event.h>
+#elif __has_include(<poll.h>)
+#include <poll.h>
 #else
 #include <sys/poll.h>
```

Comment:
Is there a platform with `<sys/poll.h>` that doesn't also have `<poll.h>`? Could we just have `<poll.h>` here?

Reply:
I'm not sure, I'm a bit afraid to touch this, as this repo may have support for OpenBSD, FreeBSD etc, but we don't have CI set up for those to verify those don't break.